index.html

```diff
@@ -6,6 +6,6 @@
     <title>csd120 - alain arseneault</title>
 </head>
 <body>
-    "Hello world!"
+    Hello world!
```

Comment:
When I ask you to insert text like `"Hello, world"` you don't need to include the quotation marks